.readthedocs.yaml

```diff
@@ -0,0 +1,21 @@
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+version: 2
+
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.12"
+  commands:
+    - asdf plugin add uv
+    - asdf install uv latest
+    - asdf global uv latest
+    - uv venv
+    - uv pip install .[docs]
+    # Generate api docs
+    - sphinx-apidoc -o $READTHEDOCS_OUTPUT/docs/api/ --no-toc --module-first
```

Comment:
Everything above this is from the scientific python template. These commands reflect the guide & nox task command provided in the template.

Currently using invoke, but I think we could switch back to nox :) That can be a separate PR.

There's an invoke task that runs these same commands.

Reply:
>Currently using invoke, but I think we could switch back to nox

Curious what your thoughts are! I think we may have already talked about why you switched to invoke, but I don't remember what you said :X

Reply:
I think the main complication was getting it to work with my conda environment. I needed to set the PYTHONPATH to the src manually, and doing that borked the nox environment somehow. I tried a few different ways but never got it to work. 

I think now that we don't really rely on conda, and that we can do a `pip install -e`, the manual PYTHONPATH setting is no longer necessary & nox should just work. Hopefully.

Reply:
That all makes sense, thanks!